website_apps_store/controllers/main.py

```diff
@@ -149,26 +149,6 @@ def shop(self, page=0, category=None, search='', ppg=False, **post):
         })
         return res
 
-    @http.route(['/shop/change_attribute_version'], type='json',
-                auth="public", website=True)
-    def change_product_attribute_version(self, **kwargs):
```

Comment:
Just to know the reason of this change. Code is supersede or just was unnecessary?

Reply:
Now uses https://github.com/OCA/apps-store/pull/67/files#diff-a1c24746e8735b6cfa432b8ef96c575fR29